enjoy/src/renderer/hooks/useConversation.tsx

```diff
@@ -87,11 +87,12 @@ export const useConversation = () => {
     });
 
     _messages
-      .sort(
-        (a, b) =>
+      .sort((a, b) => {
```

Comment:
hmm... It looks like we don't need `sort`. Maybe `reverse` is better since they are ordered `["createdAt", "DESC"]` already.

Reply:
I'm also confused after using two **sorting**.